[Custom points] Explain how to use custom points in events (#29)
GDevelopApp/GDevelop-documentation: docs/gdevelop5/objects/sprite/edit-points.md

    For example, if a sprite image is 100 x 100 pixels in dimensions, the origin of the image will lie at (0, 0) of the image, i.e. in the top-left corner of the image. The center, with respect to the origin of the image, will lie at (50, 50), i.e. (100/2, 100/2).

## Add custom points to a sprite

> The point editor can be opened with the **Edit points** button.

Reviewer: ```suggestion
The point editor can be opened with the **Edit points** button, which is found inside the Properties tab of a sprite.
```

Author: I'm not sure it's necessary to precise it.

![](open-point-editor.png){ .align-center }

> Custom points can be placed directly on the animation frame on the left and coordinates adjusted on the right.  

Reviewer: ```suggestion
Custom points can be added by clicking the **Add a point** button. The new point will be created at the top-left corner of the image, but it can be moved by dragging the point or by editing the X and Y coordinates.
```

Author: If giving each the steps is needed, maybe there is an UX issue. I prefer a short sentence that gives the general idea. It will avoid to change the text each time the UI evolves.

Reviewer: Your instructions made me think that I could simply click on the left and a new point would be created.  I actually opened up GDevelop to see if I missed that being added.  So I highly recommend my wording change.

Author: ok, I replaced "placed" by "moved".

> Custom points can be placed directly on the animation frame on the left and coordinates adjusted on the right.  

Author: ```suggestion
Custom points can be moved directly on the animation frame on the left and coordinates adjusted on the right.  
```

The **Share same points for all animations** switch can be unchecked to set different positions for each animation of the sprite.

The **Share same points for all sprites of this animation** switch can be unchecked to set deferent positions for each frame of the animation.

![](point-editor.png){ .align-center }

> ## Use custom points in events

Reviewer: ```suggestion
## Using custom points in events
```

Author: "Use" doesn't sound right?
I think it's better if all titles start with a verb in infinitive form.

> Custom points can be used in events with the `PointX` and `PointY` [expressions](/gdevelop5/all-features/expressions/).

Reviewer: ```suggestion
Custom points can be used in events with the `PointX` and `PointY` [expressions](/gdevelop5/all-features/expressions/). The name of the custom point must be surrounded by double quotation marks.
```

> Custom points can be used in events with the `PointX` and `PointY` [expressions](/gdevelop5/all-features/expressions/). The name of the custom point must be surrounded by double quotation marks.

Author: ```suggestion
Custom points can be used in events with the `PointX` and `PointY` [expressions](/gdevelop5/all-features/expressions/). Custom point names must be surrounded by double quotation marks.
```

![](custom-point-expression.png){ .align-center }

> ## Editing the default origin and center points

Reviewer: ```suggestion
## Editing the default Origin and Center points
```

Author: It's out of scope for this PR, but you can open a new one.

> If the sprite images are not centered, or the image should rotate around a specific position, you can change the position of the **Origin** and **Center** point. By default, the center point position is automatically computed. If you want to change its position, click on the Pen icon next to it:

Reviewer: ```suggestion

If the sprite needs to rotate around a specific position, such as a gun turret, this can be done by changing the position of the **Center** point.  To do this, click on the Pen icon near the coordinates of the **Center** point.

Similarly, the **Origin** point can be modified to change the starting position of an object.
```

Author: It's out of scope for this PR, but you can open a new one.
